coffee_lib/src/types/mod.rs

```diff
@@ -89,8 +89,8 @@ pub mod response {
 
     #[derive(Debug, Serialize, Deserialize)]
     pub enum UpgradeStatus {
-        UpToDate,
-        Updated,
+        UpToDate(String, String),
+        Updated(String, String),
     }
 
     #[derive(Debug, Serialize, Deserialize)]
```

Comment:
Let's add this now to make the code cleaner


```suggestion
    impl UpgradeStatus {
        pub fn date(&self) -> String {
            match self {
                UpgradeStatus::UpToDate(_, date) => date.clone(),
                UpgradeStatus::Updated(_, date) => date.clone(),
            }
        }

        pub fn commit_id(&self) -> String {
            match self {
                UpgradeStatus::UpToDate(commit_id, _) => commit_id.clone(),
                UpgradeStatus::Updated(commit_id, _) => commit_id.clone(),
            }
        }
    }
```

Reply:
Thanks!